R/pkgdown.R

```diff
@@ -1,14 +1,19 @@
 #' Use pkgdown
 #'
 #' [pkgdown](https://github.com/hadley/pkgdown) makes it easy to turn your
-#' package into a beautiful website. This helper creates `_pkgdown.yml`
-#' and `docs/` for you, and adds them to `.Rbuildignore`
+#' package into a beautiful website. This helper creates a pkgdown config file
+#' and a directory for pkgdown docs, and adds them to `.Rbuildignore`
+#'
+#' @seealso http://pkgdown.r-lib.org/articles/pkgdown.html#configuration
+#'
+#' @param config_file pkgdown yaml config file
+#' @param dst_path target directory for pkgdown docs
 #'
 #' @export
-use_pkgdown <- function() {
+use_pkgdown <- function(config_file = "_pkgdown.yml", dst_path = "docs") {
   check_is_package("use_pkgdown()")
-  edit_file(proj_path("_pkgdown.yml"))
-  use_build_ignore("_pkgdown.yml")
+  edit_file(proj_path(config_file))
+  use_build_ignore(config_file)
 
-  use_directory("docs", ignore = TRUE)
+  use_directory(dst_path, ignore = TRUE)
```

Comment:
I think we should probably build ignore the docs directory, but not create it. That would also fix #495